src/core/data-cite.js

```diff
@@ -27,6 +27,10 @@ function requestLookup(conf) {
     let href = "";
     // This is just referring to this document
     if (key.toLowerCase() === conf.shortName.toLowerCase()) {
+      console.log(
+        elem,
+        `Couldn't cite external document. The reference "${key}" is the same as the Short Name of current spec.`
```

Comment:
The remaining thing is to revise the message as the requirement is changed. My suggestion:

```suggestion
        `The reference "${key}" is resolved into the current document per \`conf.shortName\`.`
```

Reply:
Oops, already merged

Reply:
@marcoscaceres Mind a quick fix? 😀